### ISSUE TITLE
Found 2 elements with non-unique id #email: magento 2 contact

### ISSUE THREAD

**Issue by shivkumarsingh7** [magento/magento2#14850]
Getting below error on Magento 2 contact page in console.

[DOM] Found 2 elements with non-unique id #email: (More info: https://goo.gl/9p2vKq) 
```
<input name=​"email" id=​"email" title=​"Email" value class=​"input-text" type=​"email" data-validate=​"{required:​true, 'validate-email':​true}​" aria-required=​"true">​ 
<input name=​"username" id=​"email" type=​"email" class=​"input-text" data-bind=​"attr:​ {autocomplete:​ autocomplete}​" data-validate=​"{required:​true, 'validate-email':​true}​" autocomplete=​"off">​
```

<img width="1439" alt="screen shot 2018-04-25 at 7 18 12 pm" src="https://user-images.githubusercontent.com/10324098/39250090-cc92a0e2-48bd-11e8-9bc4-4186a0fb388a.png">

**Comment by julienanquetil** [comment]
Hi,
PR is done for 2.3-develop
2.2-develop is already done in commit : https://github.com/magento/magento2/commit/781ec89aaca6e6ee2853e98065416614f2edffce
Thanks

**Comment by magento-engcom-team** [comment]
Hi @shivkumarsingh7. Thank you for your report. 
The issue has been fixed in magento/magento2#14867 by @julienanquetil in 2.3-develop branch
Related commit(s):
 - [b7c0f1b6c2ddc8b2a50a1550a4a6265f8a1916e2](https://github.com/magento/magento2/commit/b7c0f1b6c2ddc8b2a50a1550a4a6265f8a1916e2)

The fix will be available with the upcoming 2.3.0 release.

**Comment by shivkumarsingh7** [comment]
@julienanquetil thank you :)

**Comment by ishakhsuvarov** [comment]
Closing this now, as it seems to be fixed in 2.3-develop and 2.2.4

**Comment by ywsw** [comment]
this is not fixed in either 2.2.4 or 2.2.5. I have 2.2.5 and I am running into the same issue. How can this be solved?

**Comment by artskeem** [comment]
this is not fixed in 2.2.5 fyi -sorry

**Comment by milossh** [comment]
@ishakhsuvarov This is still a bug in magento 2.2.5 .

**Comment by hostep** [comment]
@artskeem, @milossh: it is fixed in 2.2.4 and 2.2.5 on the contact page, I've just verified this on a vanilla installation.
Make sure you have no customisations done where it still uses a `id="email"` somewhere. Maybe you overwrite the `authentication-popup.html` file in your custom theme? Or maybe some 3rd party module inserts that somewhere?

**Comment by milossh** [comment]
@hostep I have Porto theme, with latest patches.
